lib/errors.js

```diff
@@ -24,6 +24,7 @@ class CliError extends Error {
 
         this.context = options.context || {};
         this.options = options;
+        this.help = 'Please refer to https://docs.ghost.org for troubleshooting.'
```

Comment:
https://docs.ghost.org/docs/installing-ghost-via-the-cli#troubleshooting
Can we use this link please?